### PR TITLE
add product crud, add bootstrap, admin restriction on product crud

### PR DESCRIPTION
**Pull request by varun-mfs**

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    if @product.update(product_params)
+      redirect_to @product, notice: 'Product was successfully updated.'
+    else
+      render :edit
+    end
+  end
+
+  # DELETE /products/1
+  def destroy
+    @product.destroy
```
Comment (RAJASHARMA) [inline app/controllers/products_controller.rb]: Have the else part here as well, what if destroying fails.
At least notify the user.

File: app/models/user.rb [hunk 2]
```diff
@@ -10,4 +10,8 @@ class User < ApplicationRecord
   # Roles
   CUSTOMER = 0
   ADMIN = 1
+
+  def admin?
+    role == ADMIN
```
Comment (RAJASHARMA) [inline app/models/user.rb:15]: Have the "ADMIN" in some constant and try to compare with that constant, instead of comparing hard coded value.

Comment (varun-mfs) [reply app/models/user.rb:15]: "ADMIN" is already set as a constant with a value of 1. This constant is then used in the admin? method for comparison.

Comment (RAJASHARMA) [reply app/models/user.rb:15]: Yeah Cool, It was my bad.

File: app/controllers/products_controller.rb (hunk 1 above)
Comment (varun-mfs) [reply app/controllers/products_controller.rb]: Changes are done.

**Pull request merged**